packages/components/addon/components/hds/alert/index.hbs

```diff
@@ -0,0 +1,35 @@
+<div class={{this.classNames}} role="alert" ...attributes>
```

Comment:
```suggestion
<div class={{this.classNames}} role="alert" aria-live="polite" ...attributes>
```
After doing a lot of testing on this on Friday and over the weekend, I think we need to add `aria-live="polite"` here.  Right now, our alerts are interrupting _each other_ which is not desirable behavior. 

An element with the role of `alert` implicitly has `aria-atomic="true"` and `aria-live="assertive"`. We want the `aria-atomic` attribute to stay the same so that they user has the context of the entire alert message and associated text/actions; however we want `aria-live` to be polite so it will finish waiting until the thing that is being read to the user with a screen reader is read out all the way. In this way, setting it to `polite` should at least allow the first alert message to be read out before the second alert message is read out, etc.